internal/templates/funcmap.go

```diff
@@ -53,5 +79,81 @@ func toTime(v any) string {
 	default:
 		t = time.Now()
 	}
-	return t.UTC().Format(time.RFC3339)
+	return t.UTC()
+}
+
+func formatTime(v any) string {
+	return toTime(v).Format(time.RFC3339)
+}
+
+func parseTime(v ...string) time.Time {
+	t, _ := mustParseTime(v...)
+	return t
+}
+
+func mustParseTime(v ...string) (time.Time, error) {
+	switch len(v) {
+	case 0:
+		return time.Now().UTC(), nil
+	case 1:
+		return time.Parse(time.RFC3339, v[0])
+	case 2:
+		layout := toTimeLayout(v[0])
+		return time.Parse(layout, v[1])
+	case 3:
+		layout := toTimeLayout(v[0])
+		loc, err := time.LoadLocation(v[2])
+		if err != nil {
+			return time.Time{}, err
+		}
+		return time.ParseInLocation(layout, v[1], loc)
+	default:
+		return time.Time{}, errors.New("unsupported number of parameters")
+	}
+}
+
+func toTimeLayout(fmt string) string {
+	switch strings.TrimPrefix(fmt, "time.") {
+	case "Layout":
```

Comment:
Just for convenience for the user, we could allow any case, but up to you

Reply:
Fixed with 329b3f1